front/src/SRRButton.test.js

```diff
@@ -0,0 +1,23 @@
+import React from "react";
+import SRRButton from "./SRRButton";
+import renderer from "react-test-renderer";
+
+const testFunc = () => {
+  return "Yeah, that'll do.";
+};
+
+describe("Button can be clicked", () => {
+  it("Test Button with Snapshot", () => {
```

Comment:
Small tip w.r.t. naming: the full test name will be the describe part + the it part. I'd personally name the describe part after the component ("Button") and the test name after what it is testing ("can be clicked"). As a result, the full name will be "Button can be clicked". This will show up in the snapshot as well.